src/zimcheck/zimcheck.cpp

```diff
@@ -82,11 +84,36 @@ std::string stringify(const T& x)
   return ss.str();
 }
 
-int zimcheck (const std::vector<const char*>& args)
-{
-    const int argc = args.size();
-    const char* const* argv = &args[0];
+int zimcheck(const std::map<std::string, docopt::value>& args);
+
+int zimcheck(const std::vector<const char*>& args) {
+    std::vector<std::string> args_string;
+    bool first = true;
+    for (auto arg:args) {
+        if (first) {
+            first = false;
+            continue;
+        }
+        args_string.emplace_back(arg);
+    }
 
+    docopt::Options parsed_args;
+    try {
+      parsed_args = docopt::docopt_parse(
+        USAGE,
+        args_string,
+        false,
+        false);
+    } catch (docopt::DocoptArgumentError const& error) {
+		std::cerr << error.what() << std::endl;
+		std::cout << USAGE << std::endl;
```

Comment:
Shouldn't usage be printed t stderr too?

Reply:
I took care to change the less the behavior.
Test are checking that usage is print on output. I have done the same way.